traffic_portal/test/integration/specs/Profiles.spec.ts

```diff
@@ -46,6 +46,24 @@ profiles.tests.forEach(async profilesData => {
                 await profilesPage.OpenConfigureMenu();
                 await profilesPage.OpenProfilesPage();
             });
+            profilesData.check.forEach(check => {
+                it(check.description, async () => {
+                    expect(await profilesPage.CheckCSV(check.Name)).toBe(true);
+                    await profilesPage.OpenProfilesPage();
+                });
+            });
+            profilesData.toggle.forEach(toggle => {
+                it(toggle.description, async () => {
+                    if(toggle.description.includes('hide')){
+                        expect(await profilesPage.ToggleTableColumn(toggle.Name)).toBe(false);
+                        await profilesPage.OpenProfilesPage();
+                    }else{
+                        expect(await profilesPage.ToggleTableColumn(toggle.Name)).toBe(true);
+                        await profilesPage.OpenProfilesPage();
```

Comment:
why are we re-opening the Profiles page at the end of all of these tests? These don't appear to be doing navigation, but either way a better way to reset the location after every test is with an `afterEach` call.

Reply:
I do the reopening Profiles page after each test because if the tests fail it will go back to the Profiles Page and continue with other tests.